src/uu/mv/src/mv.rs

```diff
@@ -24,9 +25,12 @@ use std::path::{Path, PathBuf};
 use uucore::backup_control::{self, BackupMode};
 use uucore::display::Quotable;
 use uucore::error::{FromIo, UError, UResult, USimpleError, UUsageError};
-use uucore::{format_usage, prompt_yes, show, show_if_err};
+use uucore::{format_usage, prompt_yes, show};
 
-use fs_extra::dir::{move_dir, CopyOptions as DirCopyOptions};
+use fs_extra::dir::{
+    get_size as dir_get_size, move_dir_with_progress, CopyOptions as DirCopyOptions,
```

Comment:
Cool! We might want to look into using that same `get_size` function in `cp` as well (in another PR).